package.json

```diff
@@ -163,7 +163,6 @@
     "metro-babel-register": "0.66.2",
     "metro-transform-plugins": "^0.66.2",
     "mkdirp": "^0.5.1",
-    "prettier": "1.19.1",
```

Comment:
From what I recall, we can actually remove a lot of the dependencies here. Keeping this scoped to prettier is fine, but I'd keep the original bug I filed open so we can remember to remove the rest.

Reply:
Actually, RN Core @ 0.68-stable has prettier listed at 2.4.1 here https://github.com/facebook/react-native/blob/4555fc7865b00f598caf86eb0e937bfac6901cc5/package.json#L155

I think you need to add this back, which might also fix CI. Once this merges, I think I'll do a larger cleanup of the package.jsons files at root and repo-config.

Reply:
I have brought prettier ^2.4.1 - but it did not seem to help with the CI builds ...